app/src/main/java/com/android/sample/model/filter/Filter.kt

```diff
@@ -20,12 +17,6 @@ open class Filter(
             UNINITIALIZED_BORN_VALUE,
             UNINITIALIZED_BORN_VALUE,
             UNINITIALIZED_BORN_VALUE),
-    var priceRange: FloatRange =
```

Comment:
**Price Range Removal:**  
   Nice call removing `priceRange`—keeps the `Filter` class clean and focused. Make sure it’s not referenced anywhere else!

Reply:
I made an attempt! As of now, apart from its mention in the Recipe, it doesn’t appear anywhere else